src/main/scala/ai/privado/languageEngine/java/passes/config/JavaPropertyLinkerPass.scala

```diff
@@ -90,20 +92,20 @@ class JavaPropertyLinkerPass(cpg: Cpg) extends PrivadoParallelCpgPass[JavaProper
   /** List of all parameters annotated with Spring's `Value` annotation, along with the property name.
     */
   private def annotatedParameters(): List[(MethodParameterIn, String)] = cpg.annotation
-    .fullName("org.springframework.*Value")
+    .name(".*Value.*")
     .filter(_.parameter.nonEmpty)
     .filter(_.parameterAssign.code("\\\"\\$\\{.*\\}\\\"").nonEmpty)
     .map { x =>
       val literalName = x.parameterAssign.code.next()
-      val value       = literalName.slice(3, literalName.length - 2)
+      val value       = Option(literalName.slice(3, literalName.length - 2)).getOrElse("")
       (x.parameter.next(), value)
```

Comment:
filter out empty Value

Reply:
There's no scope for empty value, we are filtering them out in the `generateParts` method.

Reply:
This line `val value       = Option(literalName.slice(3, literalName.length - 2)).getOrElse("")` can generate emtpy values

Reply:
Oh sorry, confused it with some other line.